cmd/pow/cmd/root.go

```diff
@@ -34,6 +36,7 @@ var (
 
 // Execute runs the root command.
 func Execute() {
+	rootCmd.SetVersionTemplate("pow build info:\n{{.Version}}\n")
```

Comment:
Custom template for printing the version to make the multi line output look better.

Reply:
Nice.